packages/shared/src/utils/location/location.js

```diff
@@ -6,3 +6,39 @@ export const getLocation = (location_list, value, type) => {
     if (location_obj) return location_obj[type];
     return '';
 };
+
+// eu countries to support
+const eu_countries = [
+    'it',
+    'de',
+    'fr',
+    'lu',
+    'gr',
+    'mf',
+    'es',
+    'sk',
+    'lt',
+    'nl',
+    'at',
+    'bg',
+    'si',
+    'cy',
+    'be',
+    'ro',
+    'hr',
+    'pt',
+    'pl',
+    'lv',
+    'ee',
+    'cz',
+    'fi',
+    'hu',
+    'dk',
+    'se',
+    'ie',
+    'im',
+    'gb',
+    'mt',
+];
+// check if client is from EU
+export const isEuCountry = country => eu_countries.includes(country);
```

Comment:
We have similar codes on the `Core` and `Cashier` it's better if you add a [Todo] for it to have only one method for `isEuCounntry` on the shared package.

Reply:
Done.